### PR TITLE
some changes

### PR DESCRIPTION
**Pull request by RomanShushakov**
Please review proposed changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
 This is an example repo
+
+Some changes were added to "REDME"
```
Comment (Iuliia-Fedkovych) [inline README.md:3]: Please correct typo in README

Comment (Iuliia-Fedkovych) [inline README.md:3]: Also maybe it's better to write README.md

**Review by Iuliia-Fedkovych — COMMENTED** [review]
Added some comments

File: README.md (hunk 1 above)
Comment (RomanShushakov) [reply README.md:3]: Ok, I will correct

Comment (RomanShushakov) [reply README.md]: Done

File: README.md [hunk 2]
```diff
@@ -1 +1,3 @@
 This is an example repo
+
+Some changes were added to README.md
```
Comment (Iuliia-Fedkovych) [inline README.md:3]: It' excessive info

**Review by Iuliia-Fedkovych — COMMENTED** [review]
One more comment

File: README.md (hunk 2 above)
Comment (RomanShushakov) [reply README.md:3]: I suppose this info is useful

Comment (Iuliia-Fedkovych) [reply README.md:3]: Ok, why not

**Pull request merged**